microovn/ovn/cli/commands.go

```diff
@@ -1,4 +1,4 @@
-package ovn
+package cli
```

Comment:
The name `cli` is a bit misleading, is this package implementing a CLI?

If the goal is to shorten the `commands` name, would `cmd` work as well?

Reply:
Thank you, yes. I had my doubts about the `cli` as well. The `cmd` alternative (with `ovn_cmd` alias usage) is much better.